### ISSUE TITLE
Config Changes Not Being Respected

### ISSUE THREAD

**Issue by Boolyman** [MJRLegends/ExtraPlanets#255]
### ---Issue Report---
**(PLEASE DONT DELETE THIS TEMPLATE, YOUR ISSUE WILL BE IGNORED AND CLOSED IF YOU DO.)**

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** [Yes or No]
Yes
### Description of Issue
[Description of the issue]
Toggling the boolean to prevent Planets and their items from being created is not working.

### Steps to Reproduce Issue
1. [First step]
**Go to config, select planets to disable, replace "true" with "false".  For instance, I am attempting to disable Titan and Titania.**
2. [Second step]
**Load game and scratch head when planet's dimension is still registered.**

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself.]
> *How do i locate a crash log on Minecraft on Curse/Twitch? see* https://gyazo.com/3c616a7dfcbe69033ea210805b093f82
> *How do i locate a crash log on Minecraft on FTB? see* http://imgur.com/a/IbWNC

**Screenshots (if applicable):** 
![2018-03-05 16_26_41-greenshot](https://user-images.githubusercontent.com/24765883/37007459-3ef1f8c6-2092-11e8-9237-0a3b3269eb76.png)

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):** [VERSION]

**0.2.6**

**Where did this bug happen:** [Single Player or Server]

**Single player.**

### Additional Information
[Any other information that may be able to help me with the problem]
 **TL;DR   I am trying to limit some of the dimensions being registered in my modpack.  I am disabling some of the ones such as Titan or Europa... but when I load the game back up, those dimensions are still there.**

**Comment by MJRLegends** [comment]
This cant be possible... are you using spongeforge there?

**Comment by Boolyman** [comment]
No just strait up forge 14.23.2.2619 on curse launcher.

**Comment by MJRLegends** [comment]
Does it show on a new world? or on galaxy map?

**Comment by Boolyman** [comment]
![2018-03-05 16_44_08-greenshot](https://user-images.githubusercontent.com/24765883/37007906-81e8abb4-2094-11e8-8779-440c79bd7f77.png)

**Comment by Boolyman** [comment]
Its a new world, yes

**Comment by MJRLegends** [comment]
As i just tried this and disabled these
![](https://i.gyazo.com/2bbdef3013ef549d9324210558761106.png)

And got only
```
[00:42:16] [main/INFO]: [CHAT] Dim -22 (Kepler 22b) : Mean tick time: 0.042 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -28 (Moon) : Mean tick time: 0.007 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -29 (Mars) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -30 (Asteroids) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -31 (Venus) : Mean tick time: 0.006 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -32 (Io) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -34 (Europa) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -35 (Deimos) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -37 (Callisto) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -38 (Ganymede) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -43 (Iapetus) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -1 (the_nether) : Mean tick time: 0.006 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -13 (Mercury) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -15 (Jupiter) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -16 (Saturn) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -17 (Uranus) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -18 (Neptune) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -19 (Pluto) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim  1 (the_end) : Mean tick time: 0.007 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -20 (Ceres) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim  0 (overworld) : Mean tick time: 4.272 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Dim -21 (Eris) : Mean tick time: 0.035 ms. Mean TPS: 20.000
[00:42:16] [main/INFO]: [CHAT] Overall : Mean tick time: 4.709 ms. Mean TPS: 20.000
``` 

i think something is wrong with a mod messing stuff up or your tps command is lying to you haha i used /forge tps but this isnt possible as that would mean java itself it broken

![](https://i.gyazo.com/5e1f716e5b7e5345dcaee61ce05ae096.png)

![](https://i.gyazo.com/e5255d5ed560431538d6a2c58020b4ee.png)
this is the registering of dimensions for moons it cant be possible, your tps command must be displaying wrong info haha

**Comment by MJRLegends** [comment]
if your still getting a issue please track down the mod messing dimensions up as sorry but this isnt a ExtraPlanets thing

**Comment by Boolyman** [comment]
Even if the TPS command was giving wrong information... that doesnt explain the planet still showing on the galaxy map.

**Comment by MJRLegends** [comment]
Your config isnt saving or being used then please delete it and regenerate everything sorry but this isnt a issue i can solve its a local issue on your machine :D

**Comment by Boolyman** [comment]
Just becuase you cant duplicate it in 5 minutes, doesnt mean its not worth addressing.  If your mod isnt compatible with another mod that may be causing this issue, should that not be something to worry about?

**Comment by MJRLegends** [comment]
Sorry this really isnt a issue with the mod. Hope you can fix what ever is causing the issue for you

**Comment by MJRLegends** [comment]
If statements must not be working if your getting this issue.... or you have made a error in the config causing it to error out, please delete the config and mod and redownload the mod and try again

**Comment by MJRLegends** [comment]
I just installed a new instance on the Twitch app these are the mods i tested with
![](https://i.gyazo.com/51a6735fc7e7085a0dcc5a86b257be85.png)

and sorry but i cant see the issue still
![](https://i.gyazo.com/66c625d83897008b6398182b510c8224.png)
![](https://i.gyazo.com/df8ba966c8b3afbd12371c5872029b70.png)
```
[00:55:55] [main/INFO]: [CHAT] Dim -22 (Kepler 22b) : Mean tick time: 0.038 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -28 (Moon) : Mean tick time: 0.006 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -29 (Mars) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -30 (Asteroids) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -31 (Venus) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -32 (Io) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -34 (Europa) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -35 (Deimos) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -37 (Callisto) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -38 (Ganymede) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -41 (Oberon) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -43 (Iapetus) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -1 (the_nether) : Mean tick time: 0.005 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -13 (Mercury) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -15 (Jupiter) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -16 (Saturn) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -17 (Uranus) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -18 (Neptune) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -19 (Pluto) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim  1 (the_end) : Mean tick time: 0.003 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -20 (Ceres) : Mean tick time: 0.004 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim  0 (overworld) : Mean tick time: 3.306 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Dim -21 (Eris) : Mean tick time: 0.018 ms. Mean TPS: 20.000
[00:55:55] [main/INFO]: [CHAT] Overall : Mean tick time: 3.589 ms. Mean TPS: 20.000
```

Sorry i really wish there was a issue i could fix for you but i dont see one

**Comment by MJRLegends** [comment]
@Boolyman Your welcome to send me a copy of your log and i will take a look but i dont see anything wrong i really wish i could help you here but its not possible for this is to happen without another factor like config erroring out due to invalid letter or entry made through user changing

**Comment by MJRLegends** [comment]
If you find a mod that is breaking this please let me know but it will be there mod that is causing the issue probably, just because i've closed this issue doesnt mean im not helping you :P Im happy to fix bugs that are ExtraPlanets sided bugs.

**Comment by Boolyman** [comment]
I'm getting this crash and the ONLY thing I change is turning the "true" to "false" on a few planets.

[crash-2018-03-05_17.33.29-client.txt](https://github.com/MJRLegends/ExtraPlanets/files/1783261/crash-2018-03-05_17.33.29-client.txt)

**Comment by MJRLegends** [comment]
which option did you disable?

**Comment by MJRLegends** [comment]
@Boolyman Please make a new issue for this crash issue, and please include a copy of the config for ExtraPlanets.

This is a different issue from the initial issue you posted here

**Comment by MJRLegends** [comment]
@Boolyman Dont worry i've found the issue, update will be out in the next 20 mins to fix it :D

**Comment by MJRLegends** [comment]
Im going to fix a few more small config issues so might be more than 20 mins :D but will be very soon
